02_calculator/calculator.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+def add(num1, num2)
+  num1 + num2
+end
+
+def subtract(num1, num2)
+  num1 - num2
+end
+
+def sum(arr)
+  count = 0
+  arr.each do |i|
+    count += i
+  end
+  count
+end
+
+def multiply(*numbers)
+  result = 1
+  numbers.each { |n| result *= n }
+  result
+end
+
+def power(num1, num2)
+  num1**num2
```

Comment:
`num1 ** num2`
also naming should full-wordy `first_number`, `other_number`, etc